baracoda/operations.py

```diff
@@ -1,18 +1,24 @@
 import logging
```

Comment:
Again although the code is readable it is not quite clear exactly what is going on. Maybe some more general documentation about the reasoning. Baracode is now our primary way of producing barcodes and could be used by others.